src/main/resources/reference.conf

```diff
@@ -24,6 +24,20 @@ akka {
         # How often to sweep out expired cache entries.
         # Note that this interval has nothing to do with TTLs
         cache-cleanup-interval = 120s
+
+        # The IPs of nameservers to use e.g.
+        # akka.io.dns.async-dns {
+        #   nameservers = ["8.8.8.8", "8.8.4.4"]
+        # }
+        nameservers = []
+
+        # Whether to use the system's /etc/resolv.conf in place
+        # of nameservers
+        resolv-conf = off
```

Comment:
Sorry @viktorklang - your request is reasonable, but I'm a little out of time on this. Would you mind if this became a separate ticket?